lib/vmpooler/api/v1.rb

```diff
@@ -499,6 +507,8 @@ def need_token!
                     arg[tag] = arg[tag].match(filter).captures.join if arg[tag].match(filter)
                   end
```

Comment:
+1 on delegating to methods with intention-revealing names.

Also, I'm not sure what `backend` refers to here as the word is pretty generic and could refer to any number of things. It could just be that I'm not familiar enough with this code, if I had to guess though I'd say it refers to the redis backend? Is that something that is presumed to be swappable with other backends like postgres?